cmd/limactl/volume.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"text/tabwriter"
+
+	"github.com/docker/go-units"
+	"github.com/lima-vm/lima/pkg/qemu"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newVolumeCommand() *cobra.Command {
+	var volumeCommand = &cobra.Command{
+		Use:   "volume",
+		Short: "Lima volume management",
+		Example: `  Create a volume:
+  $ limactl volume create data_volume
+
+  List existing volumes:
+  $ limactl volume ls
+
+  Delete a volume:
+  $ limactl volume delete data_volume`,
```

Comment:
Please update the command reference https://github.com/lima-vm/lima#command-reference

Reply:
And also needs CI
https://github.com/lima-vm/lima/blob/master/hack/test-example.sh